Makefile

```diff
@@ -204,3 +204,7 @@ upload-packages:
 .PHONY: vendor-update
 vendor-update:
 	glide update --strip-vendor
+
+.PHONY: openshiftci-presubmit-e2e
```

Comment:
shouldn't this be `openshiftci-presubmit`? @kadel

Reply:
we could have more than one presubmit job.
Other will be named something like `openshiftci-presubmit-integration`

Reply:
sorry, the next line is wrong